src/controllers/user.js

```diff
@@ -47,21 +47,13 @@ export const getById = async (req, res) => {
 
 export const getAll = async (req, res) => {
   // eslint-disable-next-line camelcase
-  const {
-    first_name: firstName,
-    cohort: inCohort,
-    cohort_id: cohortId
-  } = req.query
+  const { first_name: firstName, cohort: inCohort } = req.query
 
   const whereData = {}
   if (inCohort === 'false') {
     whereData.cohort = null
   }
 
-  if (cohortId) {
-    whereData.cohortId = Number(cohortId)
```

Comment:
Checking - is this no longer needed?

Reply:
Sean pulled from main and that code is not in main.

Also if i add it, it says cohortId is not defined.